docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -144,7 +144,7 @@ \section{Expected Technology}
 
 \section{Coding Standard}
 
-\wss{What coding standard will you adopt?}
+We aim for our project to be well documented and easy to understand. Git will be used for version control alongside GitHub for project management and CI/CD. For Git commit messages, we will follow the conventional commits specification at \url{https://www.conventionalcommits.org/en/v1.0.0/}. The provided issue templates will be used and additional templates will be added for bug reports and feature requests to ensure consistency in issue reporting. Since the TPG project is primarily written in C++, we will adopt the Google C++ Style Guide. For LaTeX documents, we will be following the file structure provided with the template and maintain consistent formatting conventions. Unit tests must be provided for all non-trivial functions and classes and integrated with the CI/CD pipeline.
```

Comment:
can we make Google C++ Style Guide to a link -> https://google.github.io/styleguide/cppguide.html